data/hydraulic-data.py

```diff
@@ -0,0 +1,129 @@
+"""Create hydraulic systems dataset"""
+
+from typing import Union, List
+from h2oaicore.data import CustomData
+import datatable as dt
+import numpy as np
+import pandas as pd
+
+class HydraulicSystemsData(CustomData):
+    # url contains hydraulic systems condition monitoring data set
+    base_url = "https://archive.ics.uci.edu/ml/machine-learning-databases/00447/"
```

Comment:
If you're only referencing a single file from this URL (data.zip), I'd suggest adding it to this URL (e.g., `...00447/data.zip`).  This will also save you a few lines of code down below because you don't need to "build" the link (i.e. `link = HydraulicSystemsData.base_url + "data.zip"`).

Reply:
Thank you for this suggestion! I removed the base_url and added the value to the link variable with "data.zip" at the end of the link.